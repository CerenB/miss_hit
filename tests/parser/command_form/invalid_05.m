% (c) Copyright 2019 Florian Schanda

function invalid_05()
    kitten = 42;
    foo = 5; % redefinition
    foo +kitten
    foo + kitten
    foo+ kitten
    foo+kitten
end
