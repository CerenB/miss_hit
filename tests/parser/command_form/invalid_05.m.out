In invalid_05.m, line 6
|     foo +kitten
|         ^^^^^^^ error: expected NEWLINE, found CARRAY instead
MISS_HIT Summary: 1 file(s) analysed, 1 error(s)
